alphastats/plots/VolcanoPlot.py

```diff
@@ -33,11 +36,18 @@
 class VolcanoPlot(PlotUtils):
     def __init__(
         self,
-        dataset,
-        group1,
-        group2,
-        column=None,
-        method=None,
+        *,
```

Comment:
Why do we need this?

Reply:
it makes people using this class downstream always use keyword arguments when creating an instance in this case

so before it was possible to have a 
```python
VolcanoPlot(mat, rawinput, ...)
```

now it makes us explicitly do 
```python
VolcanoPlot(mat=mat, rawinput=rawinput, ...)
```

Reply:
in terms of __why__ do we need this — probably to prevent errors with the order of positional arguments